### ISSUE TITLE
Dialog resize clickable area not meeting 24px a11y standards

### ISSUE THREAD

**Issue by lormanlau1** [SAP/fundamental-styles#5704]
**Describe the bug**
Clickable area for the dialog component is not 24px x 24px as stated here: https://wiki.one.int.sap/wiki/pages/viewpage.action?pageId=2866790012#ContainerResizeHandle(Horizon)-ResizeHandle. Current area is 16px x 16px

> Is this issue related to a specific component?
dialog

> What version of the Fundamental Library Styles are you using?
0.34.0 as tested but also exists on 0.38.0

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
SAP SuccessFactors

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
N/A

**To Reproduce**
1. adding fd-dialog__resize-handle class to any div
2. observe that there is a height and width of 1rem (16px)

**Expected behavior**
Ensure the target size of resize handle is 24 by 24 CSS pixels.

**Screenshots**
![Screenshot 2024-10-15 at 1 40 51 PM](https://github.com/user-attachments/assets/0d5012cf-1ffa-4f73-8fe0-48a52ae7a81a)

**Desktop (please complete the following information):**
 - OS: MacOS 15.0.1
 - Browser: Chrome
 - Version: 129.0.6668.91

**Comment by InnaAtanasova** [comment]
Hi @lormanlau1 , this has already been handled. Please check the latest fund-styles version. Thank you for reporting the issue and using the library. 

![Image](https://github.com/user-attachments/assets/46835af5-e899-48d0-b5e9-36e5b3d01ce3)
